scripts/build-docs

```diff
@@ -44,6 +44,10 @@ function buildDocsHtmlFiles() {
   buildLatestIndex $version
 }
 
+function copyDocsToLatest() {
+  cp -r "$outputDir/versions/$version/" "$outputDir/versions/latest"
+}
+
```

Comment:
You will need to also remove the steps for the`index.html` above otherwise it will still redirect to the place you don't want :)

Reply:
Do we want to keep that though or just have it go to latest if you just go to the root anyway without a version?